pkg/build/controller/build/overrides/overrides.go

```diff
@@ -98,14 +104,14 @@ func (b BuildOverrides) ApplyOverrides(pod *corev1.Pod) error {
 	return common.SetBuildInPod(pod, build)
 }
 
-func applyForcePullToPod(pod *corev1.Pod) error {
+func applyForcePullToPod(pod *corev1.Pod, pullPolicy corev1.PullPolicy) error {
```

Comment:
nit - can rename this to `applyPullPolicyToPod`

Reply:
updated in current push